network_test.go

```diff
@@ -312,63 +313,67 @@ func testNetworkMachineCNI(t *testing.T, useConfFile bool) {
 		require.NoError(t, err, "cni conf should parse")
 	}
 
-	numVMs := 10
-	vmIPs := make(chan string, numVMs)
+	if runtime.GOARCH == "arm64" {
+		t.Skip()
```

Comment:
Let's return from here. So that we don't have to nest the lines below.

https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_return_early_rather_than_nesting_deeply

Reply:
Changed the skip by replacing it with return without nesting the lines below.